README.md

```diff
@@ -24,14 +24,86 @@ Supports Node v8.3+ and modern browsers.
 
 ## Usage
 
+### CLI
+
+Spectral can be run via the command-line:
+
+```bash
+spectral lint petstore.yaml
+```
+
+Other options include:
+
+```bash
```

Comment:
Use of `bash` here results in 'random' highlighting of keywords such as `for` and `help`.